### ISSUE TITLE
FreePBX 17:  Are there any plans to upgrade the image  to Freepbx 17

### ISSUE THREAD

**Issue by hobbit378** [ugoviti/izpbx#73]
@ugoviti:
Freepbx 17 has been recently announced to be officially released in the following month (cf https://www.freepbx.org/freepbx-17-update/)
My question: Are there any plans to integrate the new FreePBX version into izPBX?
Kind Regards

**Comment by ugoviti** [comment]
Yes, when FreePBX 17 will be official released, I'll integrate it in the 20.17.x release cycle of izPBX.

Kind Regards

**Comment by hobbit378** [comment]
@ugoviti 
1. As FreePBX 17 GA  has been  [meanwhile released](https://sangomakb.atlassian.net/wiki/spaces/FP/pages/222101505/FreePBX+17) is there any update/ timeline/schedule whether an when to include in izpbx.
2. As FreePBX 17 officialy switched to Debian as base system do you plan to do so as well?
3.

**Comment by ugoviti** [comment]
I think I'll switch from Rocky Linux 8 to Debian as well.

A major migration script must be written to seamlessly accomplish this.

**Comment by psy0rz** [comment]
would it also be possible for you to make it one service per container? this might make it more clean an maintainable in the future, and you're more flexible with choosing the best distro for each component.
